docs/guide/index.md

```diff
@@ -52,6 +52,16 @@ export default defineConfig({
 
 See the list of config options in the [Config Reference](../config/)
 
+If you don't want to import vitest types in your every test file, then you can add globally `vitest/globals` to your `tsconfig.json` like this.
+
+```json
+{
+  "compilerOptions": {
+    "types": ["vitest/globals"]
+  }
+}
+```
```

Comment:
This type solely won't make it work, putting here might be misleading. I think we could move it right after the introduction of `globals: true` config

Reply:
Yeah, was about mentioning this.

Reply:
Oh right, I don't know why I did not see it there. There is even an exact tsconfig line on the Config page.
Closing PR in that case :)